test/unit/test_paired_loader.py

```diff
@@ -0,0 +1,136 @@
+"""
+Tests functionality of the PairedDataLoader
+"""
+import pytest
+
+from deepreg.dataset.loader.h5_loader import H5FileLoader
+from deepreg.dataset.loader.paired_loader import PairedDataLoader
+
+# assign values to input vars
+data_dir_path = "./data/test/h5/paired/test"
+sample_label = "sample label"
```

Comment:
the value should be "sample" or "all", check `get_label_indices` in `deepreg/dataset/util.py`